carto/utils.py

```diff
@@ -0,0 +1,22 @@
+from io import RawIOBase
+
+DEFAULT_CHUNK_SIZE = 8 * 1024  # 8 KB
```

Comment:
This constant could be inside the class, since it only really applies to that class

Reply:
True!